src/components/Layout/Header.js

```diff
@@ -14,24 +14,30 @@ function Header({ isNavbar, menuPopoverVisible, location, switchMenuPopover }) {
 
   return (
     <div className={styles.header}>
-      <Row>
-        <Col className={styles.logoCol} lg={4} md={5} sm={8} xs={12}>
-          <div className={styles.logoContainer}>
-            <img className={styles.logo} src={longhornLogo} alt="LONGHORN" />
-          </div>
-        </Col>
-        <Col lg={0} md={0} sm={16} xs={12}>
-          {isNavbar ? <div className={styles.popupMenu}>
-              <Popover placement="bottomLeft" onVisibleChange={switchMenuPopover} visible={menuPopoverVisible} overlayClassName={styles.popovermenu} trigger="click" content={<Menus {...menusProps} />}>
-                <div className={styles.button}>
-                  <Icon type="bars" />
-                </div>
-              </Popover>
-            </div> : ''}
-        </Col>
-        <Col className={styles.menuCol} lg={20} md={19} sm={0} xs={0}>
-          <Menus {...menusProps} />
+      <Row align="middle">
+        <Col className={styles.logoCol}>
+          <img src={longhornLogo} alt="LONGHORN" />
```

Comment:
We prefer not to include any branding info in the codebase, so no conditional rendering here. 
@COLDTURNIP will handle replacing the `assets/images/longhorn-logo.svg` in the prime build.